xtable-core/src/test/java/org/apache/xtable/iceberg/TestIcebergDataHelper.java

```diff
@@ -104,31 +104,50 @@ public class TestIcebergDataHelper {
       Arrays.asList(
           NestedField.optional(31, "additional_column1", Types.StringType.get()),
           NestedField.optional(32, "additional_column2", Types.LongType.get()));
+  private static final List<Types.NestedField> UUID_FIELDS =
+      Arrays.asList(NestedField.optional(33, "uuid_field", Types.UUIDType.get()));
   private static final Schema BASE_SCHEMA = new Schema(COMMON_FIELDS);
   private static final Schema SCHEMA_WITH_ADDITIONAL_COLUMNS =
       new Schema(
           Stream.concat(COMMON_FIELDS.stream(), ADDITIONAL_FIELDS.stream())
               .collect(Collectors.toList()));
-
```

Comment:
Add a schema with UUID as a separate option to avoid breaking existing test cases